CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
 Thank you for your interest in contributing to JSON Schema!  Your participation is appreciated!
 
-If your question/request/change concerns the [JSON Schema Website](json-schema.org) you are in the right place.  If your question/request/change concerns the JSON Schema specification drafts or meta-schemas, please go to [the JSON Schema specification repository](https://github.com/json-schema-org/json-schema-spec).
\ No newline at end of file
+If your question/request/change concerns the [JSON Schema Website](http://json-schema.org/) you are in the right place.  If your question/request/change concerns the JSON Schema specification drafts or meta-schemas, please go to [the JSON Schema specification repository](https://github.com/json-schema-org/json-schema-spec).
```

Comment:
Let's make this `https` instead of `http`.

```suggestion
If your question/request/change concerns the [JSON Schema Website](https://json-schema.org/) you are in the right place.  If your question/request/change concerns the JSON Schema specification drafts or meta-schemas, please go to [the JSON Schema specification repository](https://github.com/json-schema-org/json-schema-spec).
```